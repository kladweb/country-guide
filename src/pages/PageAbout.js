import React from 'react';
import { CountryAbout } from '../components/CountryAbout/CountryAbout';

export const PageAbout = () => {

  return (
    <div className='CountryAbout'>
      <div className='content'>
        <CountryAbout/>
      </div>
    </div>
  );
}