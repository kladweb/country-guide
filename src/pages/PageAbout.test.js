import React from 'react';

import renderer from 'react-test-renderer';

import { PageAbout } from "./PageAbout";

test('работа PageAbout', () => {

  // создаём тестовую версию компонента
  const component = renderer.create(
    <PageAbout/>
  );

  let componentTree = component.toJSON();
  expect(componentTree).toMatchSnapshot();

});