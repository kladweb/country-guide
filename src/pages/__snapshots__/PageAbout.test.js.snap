// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`работа PageAbout 1`] = `
<div
  className="CountryAbout"
>
  <div
    className="content"
  >
    <h1>
      ABOUT THE PROJECT
    </h1>
    <p>
      This site was created as the final project of the course "React for web application development" at the IT-ACADEMY educational center using functional components. This project was developed by Pavel Kladkevich under the guidance of the teacher Alexey Loktev.
    </p>
    <p>
      My project of the previous module "Development of Web Applications in JavaScript" with using canvas graphics:
      <span>
         
      </span>
      <a
        className="js-project"
        href="https://guess-countries.netlify.app/"
        target="_blank"
      >
        GUESS THE COUNTRY
      </a>
    </p>
  </div>
</div>
`;
